File: lib/src/modules/cart/dtos/save_to_cart_request.dart
class SaveToCartRequest {
  final int quantity;
  final int skuId;

  SaveToCartRequest({
    required this.quantity,
    required this.skuId,
  });

  Map<String, dynamic> toJson() {
    return {
      'quantity': quantity,
      'skuId': skuId,
    };
  }

}

File: src/main/java/com/example/sneakerx/dtos/cart/SaveToCartRequest.java
package com.example.sneakerx.dtos.cart;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class SaveToCartRequest {
    private Integer quantity;
    private Integer skuId;
}
